File: Python/Program/Mouse.py
from Komponen import Komponen

class Mouse(Komponen):
    def __init__(self, dpi, jenis_koneksi, merk, nama, jenis_komponen):
        super().__init__(merk, nama, jenis_komponen)
        self.dpi = dpi
        self.jenis_koneksi = jenis_koneksi

    def get_dpi(self):
        return self.dpi

    def get_jenis_koneksi(self):
        return self.jenis_koneksi

File: C++/Program/Mouse.cpp
#pragma once
#include <iostream>
#include <string>
#include "Komponen.cpp"

using namespace std;

class Mouse : public Komponen
{
private:
    int dpi; 
    string jenis_koneksi;
public:
    Mouse()
    {

    } 

    Mouse(int dpi, string jenis_koneksi, string nama, string merk, string jenis_komponen) : Komponen(nama, merk, jenis_komponen)
    {
        this->dpi = dpi;
        this->jenis_koneksi = jenis_koneksi;
    }

    void setDpi(int dpi)
    {
        this->dpi = dpi;
    }

    int getDpi()
    {
        return dpi;
    }

    void setJenis_Koneksi(string jenis_koneksi)
    {
        this->jenis_koneksi = jenis_koneksi;
    }

    string getJenis_koneksi()
    {
        return jenis_koneksi;
    }

    ~Mouse() {}
};